js/src/components/PostPublish.js

```diff
@@ -20,11 +20,15 @@ export default function PostPublish( { permalink } ) {
 			<li>
 				<a href={ "https://www.facebook.com/sharer/sharer.php?u=" + encodedUrl } target="_blank" rel="noopener noreferrer">
 					<FacebookIcon />
+					Facebook
```

Comment:
Facebook and Twitter need to be translatable strings. See https://github.com/Yoast/wordpress-seo/blob/16d4e6a484db396c4ff2fa09d082e980634fd3eb/admin/pages/social.php#L19 as an example.